### DISCUSSION TITLE
What's up with the current version of Gauguin?

### DISCUSSION THREAD

**Discussion by meikpiep** [meikpiep/gauguin#62] [Announcements]
Hi there,

the current version of Gauguin is currently three versions... Yep, no alcoholics involved here.

* Google play users can download or update to [version 0.18.0](https://github.com/meikpiep/gauguin/releases/tag/v0.18.0) released at January 5th. (Same for directly downloading the APK via this GitHub project).
* [version 0.17.0](https://github.com/meikpiep/gauguin/releases/tag/v0.17.0) from January 2nd currently gets build via F-Droid and will arrive at the next few days. There was a delay due to my fault, to name the course. This lead to
* The current version via F-Droid is [version 0.16.1](https://github.com/meikpiep/gauguin/releases/tag/v0.16.1).

Long story short: F-Droid user have to wait a bit to use 0.18.0, sorry for that.

In future, there will be an announcement when a new version gets released.

I can comment these announcements as soon as the respective version gets delivered via Google play or F-Droid, but I don't know it this helps you or it would spam you. Please leave a comment here if such comments would help you.

**Comment by meikpiep** [comment]
0.17.0 arrived via F-Droid.

**Comment by meikpiep** [comment]
0.18.0 arrived at F-Droid. Finally.
